build-logic/idea/src/main/kotlin/gradlebuild.ide.gradle.kts

```diff
@@ -81,12 +81,21 @@ fun ProjectSettings.configureCopyright() {
 
 fun ProjectSettings.configureRunConfigurations() {
     runConfigurations {
-        create<Remote>("Remote debug port 5005") {
-            mode = Remote.RemoteMode.ATTACH
+        create<Remote>("Daemon debug (port 5005)") {
+            mode = Remote.RemoteMode.LISTEN
             transport = Remote.RemoteTransport.SOCKET
             sharedMemoryAddress = "javadebug"
             host = "localhost"
             port = 5005
+            // should add auto restart here https://github.com/JetBrains/gradle-idea-ext-plugin/issues/84
```

Comment:
You could check in the xml files instead of using the API.

Reply:
Yeah, I can switch to those if the [PR](https://github.com/JetBrains/intellij-community/pull/2931) wouldn't be accepted in a reasonable time